go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/hashicorp/go-cleanhttp v0.5.2
 	github.com/hashicorp/go-multierror v1.0.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.7.0
+	github.com/huaweicloud/golangsdk v0.0.0-20210831081626-d823fe11ceba // indirect
```

Comment:
please run `go mod tidy` command as we hace change the path of golangsdk package.